zerver/lib/actions.py

```diff
@@ -4834,67 +4834,42 @@ def do_change_can_create_users(user_profile: UserProfile, value: bool) -> None:
     user_profile.save(update_fields=["can_create_users"])
 
 
-def do_change_stream_invite_only(
-    stream: Stream, invite_only: bool, history_public_to_subscribers: Optional[bool] = None
+def do_change_stream_permission(
+    stream: Stream,
+    invite_only: Optional[bool] = None,
```

Comment:
I think we should use the required kwargs pattern in Python, aka putting a `*` after the `Stream` parameter, and ask all callers to specify e.g. `invite_only=True` rather than position arguments.  This would be much more readable.

Reply:
Also, `acting_user` we generally place at the end -- it can be a required kwarg.